build.gradle

```diff
@@ -621,6 +623,7 @@ subprojects {
 
   // support for 'gradle publishToMavenLocal' etc 
   // TODO publishing test.{fvt,svt} and samples ... doesn't seem desirable? e.g., we're excluding test.{fvt,svt} jars from the tgz
+  if(!project.group.equals("edgent.platform")){
```

Comment:
Thanks for your contribution!
Rather than duplicate the "edgent.platform" specific conditionalization here, instead, how about making it more general?
```if (project.pluginManager.hasPlugin('publishing')) {```

At a higher level, I think I wish there were a way to address the problem without project-specific conditionalization in the root build.gradle.  Do you have any thoughts on an approach where specific sub projects build.gradle (like platform/*/build.gradle) could do something to override the "typical/default subproject" behavior in the root build.gradle subprojects processing?  e.g., doing something like setting a "omitPublishingPlugin" property (that the "default" subproject processing checks) or somehow nullifying the "default" publishing configuration that's created. ???